Content.Server/Ghost/GhostSystem.cs

```diff
@@ -39,9 +41,11 @@ public sealed class GhostSystem : SharedGhostSystem
         [Dependency] private readonly MobStateSystem _mobState = default!;
         [Dependency] private readonly SharedPhysicsSystem _physics = default!;
         [Dependency] private readonly IPlayerManager _playerManager = default!;
+        [Dependency] private readonly IPrototypeManager _prototype = default!;
         [Dependency] private readonly GameTicker _ticker = default!;
         [Dependency] private readonly TransformSystem _transformSystem = default!;
         [Dependency] private readonly VisibilitySystem _visibilitySystem = default!;
+        [Dependency] private readonly SharedPopupSystem _popup = default!;
```

Comment:
well now that both the client and server use this one, wouldnt it be better to make it a protected dep in the shared system